packages/suite/src/components/suite/modals/confirm/CoinmarketSavingsTerms/index.tsx

```diff
@@ -0,0 +1,104 @@
+import { Button, Icon, variables, Checkbox, H3 } from '@trezor/components';
+import React, { useState } from 'react';
+import { Translation, Modal } from '@suite-components';
+import styled from 'styled-components';
+import type { Deferred } from '@trezor/utils';
+
+const Text = styled.div`
+    padding-bottom: 20px;
+    text-align: left;
+    & + & {
+        padding-top: 20px;
+        border-top: 1px solid ${props => props.theme.STROKE_GREY};
+    }
+`;
+
+const Header = styled.div`
+    display: flex;
+    width: 100%;
+    padding: 10px 0;
+    align-items: baseline;
+    color: ${props => props.theme.TYPE_DARK_GREY};
+`;
+
+const StyledIcon = styled(Icon)`
+    padding-right: 9px;
+`;
+
+const CheckText = styled.div`
+    font-size: ${variables.FONT_SIZE.BIG};
+`;
+
+const Footer = styled.div`
+    display: flex;
+    padding: 35px 0;
+    flex: 1;
+    border-top: 1px solid ${props => props.theme.STROKE_GREY};
+`;
+
+const FooterContent = styled.div`
+    display: flex;
+    justify-content: space-between;
+    flex: 1;
+    padding: 0 35px;
+`;
+
+export type Props = {
```

Comment:
No need to fix it within this huge PR, but in general, please start prefixing Props type by Component name. It’s more explicit, easier to export type later, nicer error messages...
```suggestion
export type CoinmarketSavingsTermsProps = {
```

Reply:
Agreed.